lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,68 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
```

Comment:
These yaml entries are not going to work well as currently designed.  Let's syncup tomorrow to discuss.